src/webviews/QueryEditor/ResultPanel/IndexMetricsView.tsx

```diff
@@ -0,0 +1,159 @@
+                                            <TableCell key={column}>{column}</TableCell>
+                                        ))}
+                                    </TableRow>
+                                </TableBody>
+                            </Table>
+                        </TableHeaderCell>
+                    </TableRow>
+                </TableHeader>
+                <TableBody>
+                    {parsed.sections.map((section) => (
+                        <TableRow key={section.title}>
+                            <TableCell>{section.title}</TableCell>
+                            <TableCell>
+                                {section.indexes && section.indexes.length > 0 ? (
+                                    <Table>
+                                        <TableBody>
+                                            {section.indexes.map((cosmosdbIndex, index) => (
+                                                <TableRow
+                                                    key={index}
+                                                    {...(index === section.indexes.length - 1
+                                                        ? { style: { borderBottom: '0px' } }
+                                                        : {})}
+                                                >
+                                                    {columns.map((column) => (
+                                                        <TableCell>{cosmosdbIndex[column]}</TableCell>
+                                                    ))}
+                                                </TableRow>
+                                            ))}
+                                        </TableBody>
+                                    </Table>
+                                ) : (
+                                    '-'
+                                )}
+                            </TableCell>
+                        </TableRow>
+                    ))}
+                </TableBody>
+            </Table>
+        </>
+    );
+};
+
+/*
+Parse the indexMetricsStr and display the following information in a table format:
+Index Utilization Information
+  Utilized Single Indexes
+    Index Spec: /type/?
+    Index Impact Score: High
+    ---
+  Potential Single Indexes
+  Utilized Composite Indexes
+  Potential Composite Indexes
+  */
+
+const parseIndexMetrics = (indexMetricsStr: string): IIndexMetrics => {
+    const lines = indexMetricsStr.split('\n');
+    let title: string = '';
+    const sections: IIndexMetricsSection[] = [];
+
+    let currentSection: IIndexMetricsSection | undefined = undefined;
+    let currentIndex: { [key: string]: string } | undefined = undefined;
+
+    for (const line of lines) {
+        if (line.length === 0) {
+            continue;
+        }
+        if (line[0] !== ' ') {
+            title = line;
+            continue;
+        }
+
+        if (line.startsWith('  ') && line[2] !== ' ') {
```

Comment:
NIT:
I would use const variables like 
- const oneSpace = ' '
- const twoSpaces = "  "
- const fourSpaces = "    "

It is to hard to count them in the code and quite easy to make mistake